.github/ISSUE_TEMPLATE/where-to-report-issues.md

```diff
@@ -0,0 +1,16 @@
+---
+name: "⚠️ Where to report issues ?"
+about: File issues on the main Eclipse Che repository at https://github.com/eclipse/che/issues
+title: Issues needs to be filed on the main Eclipse Che repository
```

Comment:
Issues need~~s~~

Reply:
@azatsarynnyy fixed with other changes (Robert was reviewing it as well on che-docs repository)